test/ClangImporter/cxx_interop_ir.swift

```diff
@@ -22,3 +22,35 @@ func namespaceManglesIntoName(arg: namespacedT) {
 // CHECK: define hidden swiftcc void @"$s6cxx_ir42namespaceManglesIntoNameForUsingShadowDecl3argySo2nsV14NamespacedTypeV_tF"
 func namespaceManglesIntoNameForUsingShadowDecl(arg: NamespacedType) {
 }
+
+// CHECK-LABEL: define hidden swiftcc void @"$s6cxx_ir14accessNSMemberyyF"()
+// CHECK: %0 = call %"class.ns::T"* @{{_ZN2ns7doMakeTEv|"\?doMakeT@ns@@YAPEAVT@1@XZ"}}()
+// CHECK: call void @{{_Z4useTPN2ns1TE|"\?useT@@YAXPE?AVT@ns@@@Z"}}(%"class.ns::T"* %2)
+func accessNSMember() {
+  useT(ns.doMakeT())
+}
+
+// CHECK-LABEL: define hidden swiftcc i32 @"$s6cxx_ir12basicMethods1as5Int32VSpySo0D0VG_tF"(i8*)
+// CHECK: [[THIS_PTR1:%.*]] = bitcast i8* %0 to %TSo7MethodsV*
+// CHECK: [[THIS_PTR2:%.*]] = bitcast %TSo7MethodsV* [[THIS_PTR1]] to %class.Methods*
+// CHECK: [[RESULT:%.*]] = call i32 @_ZN7Methods12SimpleMethodEi(%class.Methods* [[THIS_PTR2]], i32 4)
+// CHECK: ret i32 [[RESULT]]
+func basicMethods(a: UnsafeMutablePointer<Methods>) -> Int32 {
+  return a.pointee.SimpleMethod(4)
```

Comment:
These are all reasonable tests but I don't see what they have to do with UsingShadowDecls…